packages/edit-site/src/components/sidebar/index.js

```diff
@@ -4,31 +4,57 @@
 import { createSlotFill } from '@wordpress/components';
 import { ComplementaryArea } from '@wordpress/interface';
 import { __ } from '@wordpress/i18n';
-import { cog, pencil } from '@wordpress/icons';
+import { cog, typography } from '@wordpress/icons';
+import { GlobalStylesControls } from '@wordpress/global-styles';
+import { useSelect } from '@wordpress/data';
 
 const { Slot: InspectorSlot, Fill: InspectorFill } = createSlotFill(
 	'EditSiteSidebarInspector'
 );
 function Sidebar() {
+	const hasGlobalStylesSupport = useSelect( ( select ) => {
+		return Object.keys( select( 'core/block-editor' ).getSettings() ).some(
```

Comment:
Could we do `return !! select( 'core/block-editor' ).getSettings(). __experimentalGlobalStylesUserEntityId;`?